LegacyCDE.Web/Web.config

```diff
@@ -19,7 +19,8 @@
   
   <appSettings>
     <!-- Quick and dirty CTS settings. -->
-    <add key="CTS_DetailedViewPagePrettyUrl" value="https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v" />
+    <add key="CTS_DetailedViewPagePrettyUrl" value="https://www.cancer.gov/research/participate/clinical-trials-search/v" />
```

Comment:
Why not update this to be a string formatter string, instead of:
`string ctViewUrl = string.Format(SearchResultsPrettyUrl + "?id={0}&r=1", cleanId.ToUpper());`

You did it below for the CTGov_RedirectionURLFormat...

Reply:
Oh, ick. I have no explanation for why that's there other than limiting the amount of working code I touched.